### PR TITLE
Improve the docs

### LINKED ISSUE #12: Create a quickstart install guide
This issue is related to creating a quickstart install guide using the automation.

- Divide the variables into multiple sections.
- Mark variables as Required or Optional.
- Improve the documentation.
- Add Contribution, DCO and Code of Conduct guides.

### PR DESCRIPTION
**Pull request by yussufsh**
Fixes #12

Signed-off-by: Yussuf Shaikh <yussuf@us.ibm.com>

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -27,7 +26,7 @@ on installing `git` for Linux and Mac.
 libvirt to accept TCP connections for remote installations. Ensure firewall rules allow remote connections to the host.
```
Comment (bpradipt) [inline README.md:26]: Possible to add a link describing how to do these ?

File: README.md [hunk 2]
```diff
@@ -27,7 +26,7 @@ on installing `git` for Linux and Mac.
...
 
 
-## Image and LPAR requirements
+# Image and LPAR requirements
```
Comment (bpradipt) [inline README.md]: How about adding details of bastion/helper node VM configuration ?

File: README.md (hunk 1 above)
Comment (yussufsh) [reply README.md:26]: I will create another doc for host setup uder docs named host-setup and will link it here in anothe PR.

File: README.md (hunk 2 above)
Comment (yussufsh) [reply README.md]: Added.

**Review by Prajyot-Parab — DISMISSED** [review]
LGTM!

**Review by bpradipt — APPROVED** [review]
lgtm

**Review by Prajyot-Parab — APPROVED** [review]

**Pull request merged**